graylog2-web-interface/src/views/components/__tests__/MigrateFieldCharts.fixtures.js

```diff
@@ -0,0 +1,53 @@
+import Immutable from 'immutable';
+
+import ViewState from 'views/logic/views/ViewState';
+import AggregationWidget from 'views/logic/aggregationbuilder/AggregationWidget';
+import AggregationWidgetConfig from 'views/logic/aggregationbuilder/AggregationWidgetConfig';
+import WidgetPosition from 'views/logic/widgets/WidgetPosition';
+
+// local storage mock for pinned-field-charts
+const mockFieldCharts = ({
```

Comment:
Maybe this file is not the right place for this mock, because it is not fix?

Reply:
I think this is fine. If you do not feel it is the right place, I would be fine with moving it to the test file itself as well.